### PR TITLE
Migrate step execution click css to spring

### PR DESCRIPTION
**Pull request by MBMIII**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -18,6 +23,26 @@
...
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
```
Comment (artemkedyk) [inline pom.xml:36]: please remove lombok dependency

File: pom.xml [hunk 2]
```diff
@@ -61,31 +86,21 @@
     <build>
         <plugins>
             <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <version>3.0.0</version>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-maven-plugin</artifactId>
+                <configuration>
+                    <excludes>
```
Comment (artemkedyk) [inline pom.xml:92]: please remove this exclusion

File: src/main/java/executor/service/services/ScenarioSourceListenerImpl.java [hunk 3]
```diff
@@ -8,20 +8,25 @@
...
 
+@Service
 public class ScenarioSourceListenerImpl implements ScenarioSourceListener {
 
     private String file;
     private ScenarioExecutor scenarioExecutor;
     private WebDriver webDriver;
 
-    public ScenarioSourceListenerImpl(String file, ScenarioExecutor scenarioExecutor, WebDriver webDriver) {
+    @Autowired
```
Comment (artemkedyk) [inline src/main/java/executor/service/services/ScenarioSourceListenerImpl.java:28]: let's remove this @Autowired annotation and use constructor injection instead

File: src/main/java/executor/service/services/step/StepExecutionClickCSS.java [hunk 4]
```diff
@@ -4,7 +4,9 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
+import org.springframework.stereotype.Component;
 
+@Component
```
Comment (artemkedyk) [inline src/main/java/executor/service/services/step/StepExecutionClickCSS.java:9]: let's remove @Component annotation and use @Service annotation here

File: src/main/java/executor/service/services/step/StepExecutionClickXpath.java [hunk 5]
```diff
@@ -4,7 +4,9 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
+import org.springframework.stereotype.Component;
 
+@Component
```
Comment (artemkedyk) [inline src/main/java/executor/service/services/step/StepExecutionClickXpath.java:9]: let's remove @Component annotation and use @Service annotation here

File: src/main/java/executor/service/services/ScenarioSourceListenerImpl.java (hunk 3 above)
Comment (MBMIII) [reply src/main/java/executor/service/services/ScenarioSourceListenerImpl.java]: @artemkedyk done

**Review by artemkedyk — APPROVED** [review]

**Pull request merged**